apps/spruce/src/pages/image/index.tsx

```diff
@@ -18,7 +24,10 @@ const Image: React.FC = () => {
 
   const selectedImage = imageId ?? firstImage;
 
-  if (!Object.values(ImageTabRoutes).includes(currentTab as ImageTabRoutes)) {
+  if (
+    !Object.values(ImageTabRoutes).includes(currentTab as ImageTabRoutes) ||
+    currentTab === undefined
```

Comment:
Just curious how the initial case wouldn't catch `currentTab === undefined`

Reply:
Tbh it should catch it... But I get a TypeScript error if I remove it because of the way `useParams` works 😔 I can swap the order of the statements so it makes a little more "sense" haha